db/schema.rb

```diff
@@ -146,6 +146,7 @@
     t.string "ensemble_identifier"
     t.text "tags"
     t.boolean "all_day", default: false
+    t.string "event_type"
```

Comment:
What is this?

Reply:
We should have types of pages so that we can tell how to render, if adding attachment link on the page, or just displaying the attachment (as in orgchart displays). I've added associated template types in application.rb but they don't seem pulled into this pr. Will add them. This gives the ability to have different types of pages with different layouts, as needed. I thought we had talked about this in the meeting with Cynthia.

Reply:
OK, this makes sense. 👍